cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -506,3 +507,56 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                         indirect=True)
+def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
+                              form_data_vm_obj_single_datastore, soft_assert):
+    """Test policy to prevent source VM from starting if migration is complete"""
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore.form_data)
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+    vm_obj = form_data_vm_obj_single_datastore.vm_list
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping.name,
+        vm_list=vm_obj,
+        start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = navigate_to(migration_plan_collection, 'All')
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True)
+
+    # wait until plan is in progress
+    wait_for(func=view.plan_in_progress, func_args=[migration_plan.name],
+        message="migration plan is in progress, be patient please",
+        delay=15, num_sec=3600)
+
+    view.migr_dropdown.item_select("Completed Plans")
+    view.wait_displayed()
+    logger.info("For plan {plan_name}, migration status : {count}, total time elapsed: {clock}"
+                .format(plan_name=migration_plan.name,
+                count=view.migration_plans_completed_list.get_vm_count_in_plan(migration_plan.name),
+                clock=view.migration_plans_completed_list.get_clock(migration_plan.name)))
+    available_tags = vm_obj.get_tags()
```

Comment:
`vm_obj` is going to be a list, you need to get an item from it and make all of these subsequent calls to a specific VM instance, not the list.

Reply:
agree! I guess I removed that while rebasing with nametuple commit, correcting it. thank you!